include/aikido/constraint/FiniteSampleable.hpp

```diff
@@ -17,15 +18,15 @@ class FiniteSampleable : public Sampleable
   /// \param _state The only sample in this constraint.
   FiniteSampleable(
       statespace::StateSpacePtr _stateSpace,
-      const statespace::StateSpace::State* _state);
+      const aikido::statespace::dart::MetaSkeletonStateSpace::ScopedState& _state);
```

Comment:
Please revert this to the original. This class is above `aikido/statespace/dart` so it shouldn't take `dart::MetaSkeletonStateSpace` component as its parameter.